MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,748 @@
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.Name == "Name")
+            {
+                e.DataFormGroupItem.IsExpanded = false;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Restricting the group expanding and collapsing
+
+The group being expanded or collapsed can be restricted by setting the [AllowExpandCollapse] to `false` in the [DataFormGroupItem].
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.Name == "Name")
+            {
+                e.DataFormGroupItem.AllowExpandCollapse = false;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Changing DataFormGroupItem visibility
+
+The [DataFormGroupItem] visibility can be changed by using the [IsVisible] property.
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.Name == "Name")
+            {
+                e.DataFormGroupItem.IsVisible = false;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the `Name` group will be hidden.
+
+## Label width customization
+
+The label and editor width can be set proportionally by using [LabelWidth] and [EditorWidth] properties of [DefaultLayoutSettings] class.
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="Sample.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm">
+        <DataForm:SfDataForm.DefaultLayoutSettings>
+            <DataForm:DataFormDefaultLayoutSettings
+                LabelWidth="1"
+                EditorWidth="2"/>
+        </DataForm:SfDataForm.DefaultLayoutSettings>
+    </DataForm:SfDataForm>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+    this.DataForm.DefaultLayoutSettings.LabelWidth = 1;
+    this.DataForm.DefaultLayoutSettings.EditorWidth = 2;
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the available width is divided into proportionally for editor (2) and label (1).
+
+
+N> Label width customization is applicable only when `LabelPosition` is Left.
+
+By default, the available width is divided equally for editor and label.
```

Comment:
6Be default available width wil be divided into 40% for label and 60% for editor

Reply:
changed content